books/free-programming-books-id.md

```diff
@@ -212,6 +213,11 @@
 * [Tutorial Belajar Framework Laravel 10](https://www.duniailkom.com/tutorial-belajar-framework-laravel/) - Duniailkom
 
 
+#### Yii
+
+* [Panduan Definitif Untuk Yii 2.0](https://www.yiiframework.com/doc/download/yii-guide-2.0-id.pdf) - Yii Software LLC. (PDF)
```

Comment:
There's another yii book in the -id list, move it here if you want to start a new section. also remove the trailing period after LLC